debian/marquee/01-certbot

```diff
@@ -2,6 +2,6 @@
 
 apt install certbot python-certbot-apache
 
-for domains in `cat DOMAINS`; do
-    certbot certonly -n --apache --agree-tos -m admin@whatwg.org -d $domains
+for domain in `cat DOMAINS`; do
```

Comment:
This should actually be plural, because it can be a value like "whatwg.org,www.whatwg.org" and documentation says "-d DOMAINS       Comma-separated list of domains to obtain a cert for"